src/components/_carousel.scss

```diff
@@ -277,4 +277,15 @@
             opacity: 1;
         }
     }
+    .thumb .privateImage {
```

Comment:
What are you trying to accomplish here? This looks like something that you should overwrite on your local component, not the repo.